src/api.js

```diff
@@ -0,0 +1,6 @@
+export async function getAPI(query) {
+  const response = await fetch(`https://bootcamp-api.codeit.kr/api${query}`);
+  const body = await response.json();

```

Comment:
이미 위에서 fetch 한 부분을 await 처리해줬기 때문에 이 부분은 await 가 의미가 없을 거 같아요!
```suggestion
  const body = response.json();
```